RetailCoder.VBE/Refactorings/ExtractInterface/ExtractInterfaceRefactoring.cs

```diff
@@ -103,7 +103,11 @@ private void AddInterface()
             {
                 interfaceComponent.Name = _model.InterfaceName;
 
-                interfaceModule.InsertLines(1, Tokens.Option + ' ' + Tokens.Explicit + Environment.NewLine);
+                var optionPresent = interfaceModule.CountOfLines > 1;
```

Comment:
CountOfDeclarationLines did something funky in the unit tests that broke them (probably related to how new modules work in the mock).  If the setting is turned on, `interfaceModule.CountOfLines` will always return 2 for a newly added module.  The `> 1` is (again) for the benefit of the mock.